core/statistics.c

```diff
@@ -326,16 +326,18 @@ void get_selected_dives_text(char *buffer, size_t size)
 
 #define SOME_GAS 5000 // 5bar drop in cylinder pressure makes cylinder used
 
-bool is_cylinder_used(struct dive *dive, int idx)
+bool is_cylinder_used(struct dive *dive, int idx, bool ignore_plannned)
```

Comment:
I think doing it inline where it's used. Test for is used and then at the one call site where it matters, also check whether its just in the planner part of the dive